src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -2,28 +2,122 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if !NETNATIVE
 extern alias System_Runtime_Extensions;
 extern alias System_Security_Principal;
+#endif
 
 using System.Diagnostics;
 using System.Collections.Generic;
 using System.Globalization;
 using System.Runtime.ConstrainedExecution;
 using Internal.Runtime.Augments;
 
+#if NETNATIVE // Temporary: these types are being added to System.Private.Corelib
```

Comment:
The content of this #if will be deleted after picking up the updated System.Private.Corelib library from ProjectN.